File: Longest_Consecutive_Sequence/submission4.py
class Solution:
    def longestConsecutive(self, arr: List[int]) -> int:
        
        n = len(arr)
        if n==0 or n==1:
            return n
        
        arr.sort()
        i = 1
        ans = 1
        sum = 1
        
        while i<n:
            if arr[i]-arr[i-1]==1:
                sum += 1
                ans = max(ans,sum)
            elif arr[i]-arr[i-1]>0:
                sum = 1
            i += 1
        
        return ans

File: Find_the_Difference/submission4.cpp
class Solution {
public:
    char findTheDifference(string s, string t) {

        char ans = 0;
        int size = s.length();

        for (int i=0; i<=size; i++){
            char character = t[i];
            ans = ans^character;
        }

        for (int i=0; i<size; i++){
            char character = s[i];
            ans = ans^character;
        }

        return ans;

    }
};